docs/src/pages/components/tree-view/tree-view.md

```diff
@@ -20,3 +20,9 @@ Tree views can be used to represent a file system navigator displaying folders a
 ### Gmail clone
 
 {{"demo": "pages/components/tree-view/GmailTreeView.js"}}
+
+## Accessibility
+
+(WAI-ARIA: https://www.w3.org/TR/wai-aria-practices/#TreeView)
+
+The component's accessibility is complete, no assistance is required.
```

Comment:
Starting of with parentheses looks weird. I would just say that it implements the behavior described in the aria practices and include the link. We don't know if it is "complete" since that isn't defined and "no assistance required" might be misunderstood. You still need assistive technology to use it.

Reply:
Thanks, I have tried to handle your feedback with `The component follows the WAI-ARIA best practices.`

Reply:
That works. Though these aren't "best" practices but "authoring" practices. Something to keep in mind in the future since "best" practices are often highly subjective (and most of the time clickbait). "Authoring practices" makes the relation to the actual aria authoring practices guide (apg) obvious.

Reply:
agree